Add defaut database seed for CC data

olohr opened the pull request:
closes #27 
Only contains initial CC data for now, random data is next

olohr committed:
Added defaut database seed for CC data

olohr committed:
Add random seed for Persons, Positions and CareerLevels

olohr committed:
Add option to clear existing data, add random connections to Person

olohr committed:
added safety for release mode

olohr commented:
There are probably smarter ways to generate random names, but it's good enough for testing

olohr committed:
fix build failing in Release mode

Yorreck reviewed (changes requested):
Please fix the bug, that appears while trying to automatically delete a person linked to an account. just dont delete this person and maybe even not their references.

Yorreck commented on appsettings.json:
```diff
@@ -17,5 +17,17 @@
+  "RandomTestSeedConfig": {
+    "Enabled": false,
+    "ClearExistingData": false,
+    "Seed": 123,
```
setting seed to null should lead to a random seed

olohr replied:
A null seed leads to a random seed, that's already implemented, or do you mean it should be the default? In that case, I agree

Yorreck reviewed (changes requested):
I setup my config like this:
"RandomTestSeedConfig": {
    "Enabled": true,
    "ClearExistingData": true,
    "Seed": 123,
    "Persons": 50,
    "Positions": 0,
    "CareerLevels": 0,
    "MemberStatus": 0,
    "PersonPositions": 30,
    "PersonCareerLevels": 150,
    "PersonMemberStatus": 150
  }

Goal was to create the default entries for status etc. an create just random persons and links. Throwing an exception cause it cant create links between person and status or else.

olohr committed:
make random seed default, add section in README

olohr committed:
don't delete account people anymore

Also add back default data after clearing the database and fix a crash caused by having no data to create connections to persons
